README.adoc

```diff
@@ -18,7 +18,7 @@
 :source-highlighter: prettify
 :page-seo-title: Open Liberty tutorial
 :page-seo-description: Find out how to run microservices on Open Liberty 
-= Getting started with the Open Liberty application server
+= Deploying and packaging apps
```

Comment:
Can we spell out the word "apps"? Also with the title change, the abstract and the `what you'll learn` sessions need to be adjusted according too. @Kubik42

Reply:
I'm fine with changing apps to applications.